single-point-processing-example/index.js

```diff
@@ -1,13 +1,44 @@
+const { RekognitionClient, DetectLabelsCommand } = require("@aws-sdk/client-rekognition")
 
 /**
  * Used to identify if a PNG is a dog or cat
  *
  * @param {String} bucket_name name of S3 bucket
  * @param {String} key key name within bucket (including prefix)
- * @returns {String} string of dog or cat
+ * @returns {String} string of "Dog", "Cat" or "Neither"
  */
-const dogOrCat = (bucketName, key) => {
-  return "dog"
+const dogOrCat = (bucket_name, key) => {
```

Comment:
This function should be async, that's probably why the console.log is sometimes not showing

Reply:
Why should this be async? (JS noob talking).

Is it because the AWS SDK calls are async so anything wrapping it must be too?